res/controllers/Traktor Kontrol S4 MK3.hid.xml

```diff
@@ -9,6 +9,615 @@
+                    <value label="Artist">1</value>
+                    <value label="Title">2</value>
+                    <value label="Album">3</value>
+                    <value label="Albumartist">4</value>
+                    <value label="Year">5</value>
+                    <value label="Genre">6</value>
+                    <value label="Composer">7</value>
+                    <value label="Grouping">8</value>
+                    <value label="Tracknumber">9</value>
+                    <value label="Filetype">10</value>
+                    <value label="Native Location">11</value>
+                    <value label="Comment">12</value>
+                    <value label="Duration">13</value>
+                    <value label="Bitrate">14</value>
+                    <value label="BPM">15</value>
+                    <value label="ReplayGain">16</value>
+                    <value label="Datetime Added" default="true">17</value>
+                    <value label="Times Played">18</value>
+                    <value label="Rating">19</value>
+                    <value label="Key">20</value>
+                    <description>Fifth column when iterating over sortable library columns using VIEW+Library Encoder press</description>
+                </option>
+                <option
+                    variable="librarySortableColumns6Value"
+                    type="enum"
+                    label="Sixth column">
+                    <value label="---" default="true">0</value>
+                    <value label="Artist">1</value>
+                    <value label="Title">2</value>
+                    <value label="Album">3</value>
+                    <value label="Albumartist">4</value>
+                    <value label="Year">5</value>
+                    <value label="Genre">6</value>
+                    <value label="Composer">7</value>
+                    <value label="Grouping">8</value>
+                    <value label="Tracknumber">9</value>
+                    <value label="Filetype">10</value>
+                    <value label="Native Location">11</value>
+                    <value label="Comment">12</value>
+                    <value label="Duration">13</value>
+                    <value label="Bitrate">14</value>
+                    <value label="BPM">15</value>
+                    <value label="ReplayGain">16</value>
+                    <value label="Datetime Added">17</value>
+                    <value label="Times Played">18</value>
+                    <value label="Rating">19</value>
+                    <value label="Key">20</value>
+                    <description>Sixth column when iterating over sortable library columns using VIEW+Library Encoder press</description>
+                </option>
+            </row>
+        </group>
+        <group label="Jog wheel">
+            <option
+                variable="loopWheelMoveFactor"
+                type="integer"
+                min="20"
```

Comment:
make sure the min and max are bigger than is reasonable, just in case someone wants to do something wild :)

Reply:
yeah probably a bigger max.  I have seen some videos of djs cranking the CDJ wheel sensitivity way way up and getting some really cool effects.    Sure let's try 10x!